practices/software_development_kit.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: software-development-kit
+stages:
+  - development
+short_description: A SDK is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so.
+tags:
+  - sdk
+  - software development
+  - software engineering
+  - development
+  - developer experience
+keywords:
+  - product sprint
+  - software development
+  - software engineering
+  - product development
+  - debugging
+  - documentation
+  - libraries
+  - apis
+  - sdk
+---
+
+# Software Development Kit (SDK)
+
+**TL;DR**
+
+It is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so. You have what you need to get the job done, and what's included in your kit toolbox, varies from platform to platform.
+
+## What is an SDK
+
+A Software Development Kit (SDK) is the toolbox of the software development world. But instead of tools like screwdrivers and pliers, SDKs contain libraries and Application Programming Interfaces (APIs) that make it easier to create applications for specific software and/or hardware.
+
+It can also be thought of as a wrapper library developed by a service provider which provides an abstraction around the services or capabilites that the platform has or supports.
+
+![SDK](/files/software_development_kit.png)
+[CleverTap: What is an SDK? Everything You Need to Know](https://clevertap.com/blog/what-is-an-sdk/)
+
+### Why You Might Want an SDK
+
```

Comment:
Please, this part should explain why someone should have an SDK. What are the benefits of it? You are explaining what SDK is but I don't see why I should ask my developers to spend their time on creating one.